apps/meteor/app/api/server/v1/cloud.ts

```diff
@@ -122,3 +123,23 @@ API.v1.addRoute(
 		},
 	},
 );
+
+API.v1.addRoute(
+	'cloud.syncWorkspace',
+	{
+		authRequired: true,
+		permissionsRequired: ['view-privileged-setting'],
+		rateLimiterOptions: { numRequestsAllowed: 2, intervalTimeInMS: 60000 },
+	},
+	{
+		async get() {
```

Comment:
this looks more like a `POST`.. is there a reason for it to be a `GET`?

Reply:
I thought that the `GET` would fit better in that scenario, since just performs an action instead of saving data.
Why do you think that a `POST` would be better?

Reply:
> GET requests are only used to request data (not modify)

causing an effect or changing an internal data, even without sending any new data is still a change so  `POST` suits better

Reply:
Got it!  Thanks